freebsd/FreeBSDProcessList.c

```diff
@@ -442,6 +442,13 @@ void ProcessList_goThroughEntries(ProcessList* super, bool pauseProcessUpdate) {
       return;
    }
 
+   int ccpu;
+   size_t size = sizeof(ccpu);
+   if (sysctlbyname("kern.ccpu", &ccpu, &size, NULL, 0) == -1) {
+      ccpu = 0;
+   }
+   const double decayfactor = log(ccpu / kernelFScale);
```

Comment:
`kernelFScale` might be zero. Not ensured non-zero by the code querying this from the syscall.

Reply:
https://github.com/htop-dev/htop/blob/feec16cbb53dabc6a52ef2f69a6a13798be82617/freebsd/FreeBSDProcessList.c#L138-L142

You mean `kern.fscale` might be `0`? Is that possible?

Reply:
Highly unlikely, given that there's other code already that does a similar division and I'm not aware of any bug reports regarding this. Mostly DiD note.

Reply:
FYI, https://is.cuni.cz/webapps/zzp/download/120118301

Reply:
Actually just looking at the remarked upon code part: `ccpu` is set to `0`, which will cause a math error with the following calculation for `decayfactor`. I don't think that's right?

Reply:
I've made pull request #1275 that would reject zero `kern.fscale` value.